src/docs/zksync-era.mdx

```diff
@@ -81,12 +81,12 @@ links:
                 {
                     label: 'Latency',
                     tooltip: 'The time it takes for a message to be made available on Ethereum after being included in a zkSync Era block and sequenced to L1.',
-                    value: 'To be added',
+                    value: '24 hours',
                 },
                 {
                     label: 'Cost',
                     tooltip: 'The gas cost of proving and executing a message on Ethereum after being sent from zkSync Era',
-                    value: 'To be added',
+                    value: 'No added cost',
```

Comment:
We would need to estimate the gas of calling the function within the zk sync era contract that returns information on whether the message is valid or no